test/liquidate/liquidated-cluster.ts

```diff
@@ -123,4 +123,46 @@ describe('Liquidate Tests', () => {
     await utils.progressBlocks(2);
     expect(await ssvViews.getBalance(helpers.DB.owners[1].address, clusterEventData.operatorIds, clusterEventData.cluster)).to.be.equals(minDepositAmount * 2 - burnPerBlock * 2);
   });
+
+  it('Remove validator -> withdraw -> liquidate', async () => {
```

Comment:
I dont understand how this test passes with error saying ClusterNotLiquidatable (based on my commecnt above it should be an error), im assuming its getting this error not because of 0 validators but rather balance is not low enough.

Reply:
Yes, let me change the title of the test to be more clear.
In the last step, when trying to liquidate, the cluster has this state:
Balance: 157000000
Validators: 0

the minimum Liquidation Collateral is 200000000, greater than the cluster balance so it could be liquidatable, but with the current condition, as it has zero validators, it is not liquidated.

Reply:
understood i just thought that the other code should for sure fail so i guessed maybe you had a flase positive going but awesome!